common/app/model/content.scala

```diff
@@ -334,6 +334,7 @@ object Article {
     val javascriptConfig: Map[String, JsValue] = Map(
       ("contentType", JsString(contentType)),
       ("isLiveBlog", JsBoolean(content.tags.isLiveBlog)),
+      ("isMinuteArticle", JsBoolean(content.tags.isUSMinuteSeries)),
```

Comment:
A minute article is along the same lines (template/content wise) as a live blog - it is designed as a new template but the format, in terms of frontend templates, css, js, composer (eventually), will be at the same level of difference from an article as a liveblog. With that in mind, what are the guidelines / advantages of using `conditionalConfig` for this and what makes that usage make sense in this case but not with  `isLiveBlog`?